substrate/frame/asset-rate/src/lib.rs

```diff
@@ -155,18 +156,18 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::create())]
 		pub fn create(
 			origin: OriginFor<T>,
-			asset_kind: T::AssetKind,
+			asset_kind: Box<T::AssetKind>,
```

Comment:
there is a separate PR for this https://github.com/paritytech/polkadot-sdk/pull/1545